js/trickleButton.js

```diff
@@ -0,0 +1,8 @@
+import Adapt from 'core/js/adapt';
+import TrickleButtonView from './TrickleButtonView';
+import TrickleButtonModel from './TrickleButtonModel';
```

Comment:
```suggestion
import TrickleButtonView from './trickleButtonView';
import TrickleButtonModel from './trickleButtonModel';
```

Reply:
I've renamed the files instead. They're classes, they should be capitalised.